packages/simulation-sdk-wagmi/test/hooks/useSimulationState.test.ts

```diff
@@ -289,6 +289,13 @@ describe("useSimulationState", () => {
     client,
     expect,
   }) => {
+    const amount = 1_000000n;
+
+    await client.approve({
+      address: usdc,
+      args: [morpho, amount],
+    });
```

Comment:
what does it change?

Reply:
Was failing with "insufficient allowance" insteaf of "insufficient balance"